nameko/amqp/__init__.py

```diff
@@ -0,0 +1,4 @@
+# backwards compat imports
+from .utils import verify_amqp_uri  # noqa: F401
```

Comment:
They're "unused imports" otherwise. I'm only pulling them into this package for a backwards compatibility import of `from nameko.amqp import verify_amqp_uri`